xterm/index.cgi

```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+
+# Switch to given user
```

Comment:
How is this change related to bundling dependencies? Please separate it out ..

Reply:
Will do.